resources/public/css/style.css

```diff
@@ -0,0 +1,52 @@
+/* some style */
```

Comment:
I did use Twitter Bootstrap, but just included it in a script tag from a cdn. When included with some backend, then it would need to be integrated integrated with the other assets.